### PR TITLE
[1221][IMP] Move auth page texts to website config settings

### PR DESCRIPTION
**Pull request by TimLai125**
#[1221](https://www.quartile.co/web?debug=#id=1221&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: website_auth_view_mi7/i18n/ja.po [hunk 1]
```diff
@@ -16,9 +16,21 @@ msgstr ""
 "Plural-Forms: \n"
 
 #. module: website_auth_view_mi7
-#: model:ir.ui.view,arch_db:website_auth_view_mi7.signup
-msgid ". If you agree to the Membership Terms, click the checkbox on the left and you can further proceed with the signup."
-msgstr "をよくお読みの上、全ての項目に同意頂ける場合は、左のチェックボックスにチェックをいれ、下の「アカウントを作成」ボタンを押してください。"
+#: model:ir.ui.view,arch_db:website_auth_view_mi7.custom_signup_layout
+msgid "<span style=\"color:red\">*</span> Required fields."
+msgstr "<span style=\"color:red\">*</span> 入力必須のフィールド"
```
Comment (yostashiro) [inline website_auth_view_mi7/i18n/ja.po:21]:
```suggestion
msgstr "<span style=\"color:red\">*</span> 入力必須項目"
```

File: website_auth_view_mi7/i18n/ja.po [hunk 2]
```diff
@@ -16,9 +16,21 @@ msgstr ""
...
+
+#. module: website_auth_view_mi7
+#: model:ir.model.fields,field_description:website_auth_view_mi7.field_website_config_settings_login_page_header_text
+#: model:ir.model.fields,field_description:website_auth_view_mi7.field_website_login_page_header_text
+msgid "Login Page Header Text"
+msgstr "ログインページのヘッダテキスト"
```
Comment (yostashiro) [inline website_auth_view_mi7/i18n/ja.po]:
```suggestion
msgstr "ログインページヘッダーテキスト"
```

File: website_auth_view_mi7/i18n/ja.po [hunk 3]
```diff
@@ -16,9 +16,21 @@ msgstr ""
...
+
+#. module: website_auth_view_mi7
+#: model:ir.model.fields,field_description:website_auth_view_mi7.field_website_config_settings_login_page_signup_text
+#: model:ir.model.fields,field_description:website_auth_view_mi7.field_website_login_page_signup_text
+msgid "Login Page User Register Text"
+msgstr "ログインページのアカウント作成テキスト"
```
Comment (yostashiro) [inline website_auth_view_mi7/i18n/ja.po]:
```suggestion
msgstr "ログインページユーザ登録テキスト"
```

File: website_auth_view_mi7/i18n/ja.po [hunk 4]
```diff
@@ -31,9 +43,10 @@ msgid "My Page"
 msgstr "マイページ"
 
 #. module: website_auth_view_mi7
-#: model:ir.ui.view,arch_db:website_auth_view_mi7.signup
-msgid "Please read and understand the"
-msgstr " "
+#: model:ir.model.fields,field_description:website_auth_view_mi7.field_website_config_settings_password_reset_page_header_text
+#: model:ir.model.fields,field_description:website_auth_view_mi7.field_website_password_reset_page_header_text
+msgid "Password Reset Page Header Text"
+msgstr "パスワードリセットのページヘッダテキスト"
```
Comment (yostashiro) [inline website_auth_view_mi7/i18n/ja.po]:
```suggestion
msgstr "パスワードリセットページヘッダーテキスト"
```

File: website_auth_view_mi7/i18n/ja.po [hunk 5]
```diff
@@ -57,31 +70,29 @@ msgid "Sign up"
 msgstr "アカウントを作成"
 
 #. module: website_auth_view_mi7
-#: model:ir.ui.view,arch_db:website_auth_view_mi7.custom_login_layout
-msgid "Text paragraph. Text paragraph. Text paragraph. Text paragraph. Text paragraph. Text paragraph."
-msgstr "テキスト/テキスト/テキスト/テキスト/テキスト/テキスト"
+#: model:ir.model.fields,field_description:website_auth_view_mi7.field_website_config_settings_signup_page_header_text
+#: model:ir.model.fields,field_description:website_auth_view_mi7.field_website_signup_page_header_text
+msgid "Signup Page Header Text"
+msgstr "アカウント作成ページのヘッダテキスト"
```
Comment (yostashiro) [inline website_auth_view_mi7/i18n/ja.po]:
```suggestion
msgstr "サインアップページヘッダーテキスト"
```

File: website_auth_view_mi7/i18n/ja.po [hunk 6]
```diff
@@ -57,31 +70,29 @@ msgid "Sign up"
...
 
 #. module: website_auth_view_mi7
-#: model:ir.ui.view,arch_db:website_auth_view_mi7.custom_signup_layout
-msgid "お客様の新規アカウントを作成します。Text paragraph. Text paragraph. Text paragraph. Text paragraph. Text paragraph. Text paragraph."
-msgstr "お客様の新規アカウントを作成します。テキスト/テキスト/テキスト/テキスト/テキスト/テキスト"
+#: model:ir.model.fields,field_description:website_auth_view_mi7.field_website_config_settings_signup_page_terms_text
+#: model:ir.model.fields,field_description:website_auth_view_mi7.field_website_signup_page_terms_text
+msgid "Signup Page Terms and Conditions Text"
+msgstr "アカウント作成ページの会員規約テキスト"
```
Comment (yostashiro) [inline website_auth_view_mi7/i18n/ja.po]:
```suggestion
msgstr "サインアップページ会員規約テキスト"
```

File: website_auth_view_mi7/models/website.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    _inherit = "website"
+
+    login_page_header_text = fields.Html("Login Page Header Text", translate=True,)
+    login_page_signup_text = fields.Html(
+        "Login Page User Register Text", translate=True,
+    )
+    signup_page_header_text = fields.Html("Signup Page Header Text", translate=True,)
+    signup_page_terms_text = fields.Html(
+        "Signup Page Terms and Conditions Text", translate=True,
+    )
```
Comment (yostashiro) [inline website_auth_view_mi7/models/website.py]: No need to add `sanitize=False`? (Same for other fields.)

**Review by yostashiro — CHANGES_REQUESTED** [review]

File: website_auth_view_mi7/i18n/ja.po (hunk 1 above)
Comment (TimLai125) [reply website_auth_view_mi7/i18n/ja.po:21]: This translation here follows the requirements from the customer, therefore I will keep "フィールド"

**Review by yostashiro — APPROVED** [review]
LGTM!

**Pull request merged**